src/main/java/org/opentripplanner/graph_builder/module/osm/OpenStreetMapModule.java

```diff
@@ -104,34 +104,28 @@ public class OpenStreetMapModule implements GraphBuilderModule {
    * Allows for arbitrary custom naming of edges.
    */
   public CustomNamer customNamer;
-
```

Comment:
Did you delete the blank lines here? If I insert/remove blank lines the `prettier formatter` leaves them in - it does not change them.

We usually uses a blank line before all JavaDoc comments. So, for the old style classes with public fields we do add blank lines between the fields. As an improvement we try to migrate to private fields, and then move the JavaDoc to the accessor. Then all filed are listed without any blank lines between.

Reply:
Prettier for intellij (via nodejs) does this. Sadly it's different from the Maven command line version. :cry: